students/sangyeon/users/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+class User(models.Model): 
+    name         = models.CharField(max_length=50)
+    email        = models.EmailField(max_length=50, unique=True)
+    password     = models.CharField(max_length=100)
+    phone_number = models.CharField(max_length=50)
+    create_at    = models.DateTimeField(auto_now_add=True)
+    updated_at   = models.DateField(auto_now=True)
```

Comment:
```suggestion
    updated_at   = models.DateTimeField(auto_now=True)
```

`updated_at` column 또한 `DateTimeField`로 수정해주세요! 
추가로 DateField 와 무슨 차이가 있는지도 찾아보시면 좋을 것 같습니다. (그래서 왜 DateTimeField를 쓰는지도요!)

최종적으로 수정되면 아래 명령어를 통해 migration 파일들 같이 올라오게 push 해주세요!
```
python manage.py makemigrations
```

Reply:
수정과 migration 추가 완료했습니다!